### ISSUE TITLE
Is this still maintained? 

### ISSUE THREAD

**Issue by schliffen** [Ploppz/blackbox#1]
Is there any updates since 2020?

**Comment by Ploppz** [comment]
No, I used it for a project back then and although it's limited, I tried to envision a framework that could support many different search methods. But I don't have any use for it at this moment personally so I'm not working on it.
